tutorials/time-freq/ssvep.py

```diff
@@ -134,15 +136,20 @@
 
 
 ###############################################################################
-# Calculate SNR
-# ^^^^^^^^^^^^^
+# Calculate signal to noise ration (SNR)
```

Comment:
```suggestion
# Calculate signal to noise ratio (SNR)
```

You are the third person I know that makes this typo (ratio -> ration)!

Reply:
I am one of the first two :)

Reply:
thanks for spotting!
don't remember me doing this typo before :D 
'n' is next to the spacebar on my keyboard layout, maybe thats the reason..